### ISSUE TITLE
Pull downs should match dark theme. 

### ISSUE THREAD

**Issue by smolinari** [dreamonkey/quasar#76]
I think these menus should follow the dark theme design. 

![image](https://user-images.githubusercontent.com/3314957/153208603-df92a344-159a-461c-b7d3-4bc13c27905c.png)

![image](https://user-images.githubusercontent.com/3314957/153208930-07c5b80f-cc5b-455e-9545-3434f1ec577b.png)

Scott

**Comment by IlCallo** [comment]
I think we checked this and the dark mode on dropdowns wasn't pretty, but we can try it again

**Comment by NiccoloMenozzi** [comment]
We did not design the website in order to get to a dark/light mode approach. The original goals agreed with Razvan were focusing on branding necessities and so we opted for something that looks different from usual. We are trying to express a strong identity and details like this could help.
We agree that dark/light mode will be one of the thing to improve but before doing so we need to consolidate a lot of basic brand guidelines for a Quasar design system which at the moment we do not have the time to think about and design properly. We will get there but according to our roadmap is not a priority and as soon as the website will be ready we will begin to discuss what comes next design-side, hopefully with the help of Razvan as well.
